packages/search/src/users/errors.ts

```diff
@@ -0,0 +1,11 @@
+export class GraphQLError extends Error {
```

Comment:
I am conflicted over whether this class is necessary. On one hand, we could just throw what the GraphQL API gives us. On the other hand, it would be nice if we abstracted that implementation a bit to the client and gave the client an easier-to-use error message